src/settings-ui/Settings.UI/App.xaml.cs

```diff
@@ -263,12 +263,23 @@ public static ElementTheme SelectedTheme()
             }
         }
 
+        public static string SelectedFlyoutTheme()
+        {
+            return SettingsRepository<GeneralSettings>.GetInstance(settingsUtils).SettingsConfig.FlyoutTheme.ToUpper(CultureInfo.InvariantCulture);
+        }
+
         public static bool IsDarkTheme()
         {
             var selectedTheme = SelectedTheme();
             return selectedTheme == ElementTheme.Dark || (selectedTheme == ElementTheme.Default && ThemeHelpers.GetAppTheme() == AppTheme.Dark);
         }
 
+        public static bool IsFlyoutDarkTheme()
+        {
+            var selectedTheme = SelectedFlyoutTheme();
+            return selectedTheme == "DARK" || (selectedTheme == "SYSTEM" && ThemeHelpers.GetSystemTheme() == AppTheme.Dark);
```

Comment:
```suggestion
            return selectedTheme == ElementTheme.Dark || (selectedTheme == "SYSTEM" && ThemeHelpers.GetSystemTheme() == AppTheme.Dark);
```

And probably `ElementTheme.Default` for system too.

Reply:
Done. This was something I have had as a local change for a while while experimenting. It has now been committed.